html-forms/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div>
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <div>
+      <input type="text" id="email" name="email">
+    </div>
+  </div>
+  <div>
+    <div>
+      <label for="codeCool">Why is coding cool?</label>
+    </div>
+    <div>
+      <select id="codeCool" name="codeCool">
+        <option value="">Select an option</option>
+        <option value="interesting">Because its interesting</option>
+        <option value="fun">Because its fun</option>
+        <option value="powerful">Because its powerful</option>
+      </select>
+    </div>
+  </div>
+  <div>
+    <div>
+      <div>
+        Favorite Language (choose one)
+      </div>
+      <label for="html"> </label>
+      <input type="radio" name="html" id="html" value="html" />HTML
+      <label for="css"></label>
+      <input type="radio" name="css" id="css" value="css" />CSS
+      <label for="js"></label>
+      <input type="radio" name="js" id="js" value="js"/>Javascript
```

Comment:
For the radio buttons to work properly they need to have all the same `name` attribute. This makes it so only one radio can be selected at a time.